### PR TITLE
[DOC] More on cp_r

### PR DESCRIPTION
**Pull request by BurdetteLamar**
I have not provided an example for cp_r with multiple dirpaths in +src+ b/c it would be huge.  Should I add one?

  If +src+ is an array of paths and +dest+ is a directory,
  recursively copies from each path in +src+ to +dest+;
  the paths in +src+ may point to files and/or directories.

### CODE REVIEW

File: lib/fileutils.rb [hunk 1]
```diff
@@ -841,31 +842,53 @@ def cp(src, dest, preserve: nil, noop: nil, verbose: nil)
...
-  #   File.exist?('dest3/src3/dir0/src1.txt') # => true
-  #   File.exist?('dest3/src3/dir1/src2.txt') # => true
-  #   File.exist?('dest3/src3/dir1/src3.txt') # => true
+  #   tree('dest3')
+  #   dest3
+  #   `-- src3
+  #     |-- dir0
+  #     |   |-- src0.txt
+  #     |   `-- src1.txt
+  #   `-- dir1
```
Comment (peterzhu2118) [inline lib/fileutils.rb:885]: I think this line's indentation is off by one.

File: lib/fileutils.rb [hunk 2]
```diff
@@ -817,7 +817,8 @@ def cp(src, dest, preserve: nil, noop: nil, verbose: nil)
   alias copy cp
   module_function :copy
 
-  # Recursively copies files from +src+ to +dest+.
+  # Recursively copies files from +src+ (a single path or an array of paths)
+  # to +dest+ (a single path).
   #
   # Arguments +src+ and +dest+
```
Comment (peterzhu2118) [inline lib/fileutils.rb:823]: For consistency, I think we should specify here that `src` is a single or an array of paths and `dest` is a single path.

Comment (peterzhu2118) [reply lib/fileutils.rb:823]: The docs for this method says:

> Recursively copies files from +src+ (a single path or an array of paths)
> to +dest+ (a single path).
> 
> Arguments +src+ and +dest+
> should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].

Whereas, for example, the docs for `ln_s` says:

> Creates {symbolic links}[https://en.wikipedia.org/wiki/Symbolic_link].
>
> Arguments +src+ (a single path or an array of paths)
> and +dest+ (a single path)
> should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].

So yeah, perhaps we can remove duplication in the docs.

File: lib/fileutils.rb (hunk 1 above)
Comment (BurdetteLamar) [reply lib/fileutils.rb]: Fixed.

**Comment by BurdetteLamar** [comment]
@peterzhu2118, I have not provided an example for cp_r with multiple dirpaths in +src+ b/c it would be huge. Should I add one?

**Comment by peterzhu2118** [comment]
I agree. I don't think it's necessary to have an example for that.

**Review by peterzhu2118 — APPROVED** [review]

**Pull request merged**